controllers/comments.controllers.js

```diff
@@ -28,3 +30,13 @@ exports.postCommentByArticleId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  Promise.all([checkCommentExists(comment_id), removeCommentById(comment_id)])
```

Comment:
we dont need a checkCommentexists because if it doesnt the rows will return 0 and its caught by your Promise.reject :)